src/framework/preferences/js/SeparatedPanelPrefsEditor.js

```diff
@@ -217,7 +257,12 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             var iframeWindow = that.iframe[0].contentWindow;
             that.iframeDocument = iframeWindow.document;
 
-            that.jQuery = iframeWindow.jQuery;
+            // Use the iframe's own jQuery if it's loaded, otherwise use the
+            // parent document
+
+            var iframeWindowHasJQuery = iframeWindow.jQuery !== undefined;
+            that.jQuery = iframeWindowHasJQuery ? iframeWindow.jQuery : $;
```

Comment:
This is probably safer. Or, per @amb26's comments below about implementation switches based on conditionals, maybe it should be parameterised in the `fluid.prefs.separatedPanel.renderIframe` based on the same contextAwareness checks.